lib/system_description.rb

```diff
@@ -61,10 +61,14 @@ def load!(name, store, options = {})
     # loading of the system description succeeds.
     def load(name, store, options = {})
       manifest = Manifest.load(name, store.manifest_path(name))
-      manifest.validate
+      if !options[:skip_validation]
```

Comment:
Please use a trailing if clause when the statement is short and there are no other branches.